pkg/lib/lastlog2.py

```diff
@@ -0,0 +1,22 @@
+# Print lastlog2 database as JSON; used in ./login.js
+import json
+import os
+import sqlite3
+import sys
+
+DB = "/var/lib/lastlog/lastlog2.db"
+
+if not os.path.exists(DB):
+    sys.exit(f"{DB} does not exist")
+
+con = sqlite3.connect(DB)
+cur = con.cursor()
+query = "SELECT Name, Time, TTY, RemoteHost FROM Lastlog2"
```

Comment:
Is the schema documented somewhere?

Reply:
I just got it from `.schema` in sqlite3. The [manpage](https://man7.org/linux/man-pages/man3/lastlog2.3.html) doesn't document it. Otherwise, it's just [in the upstream source code](https://github.com/util-linux/util-linux/blob/master/liblastlog2/src/lastlog2.c#L239).

The format doesn't look prone to changes, but of course it could happen in principle. Then our tests should pick it up and we need to deal with it.